images/linux/ubuntu1604.json

```diff
@@ -194,6 +194,7 @@
                 "{{template_dir}}/scripts/installers/azure-cli.sh",
                 "{{template_dir}}/scripts/installers/azure-devops-cli.sh",
                 "{{template_dir}}/scripts/installers/basic.sh",
+                "{{template_dir}}/scripts/installers/bicep.sh",
```

Comment:
Please, also remove changes for Ubuntu 16.04. It is currently under deprecation process, we do not update it anymore, and will remove on September, 20. https://github.com/actions/virtual-environments/issues/3287